src/registrar/templates/domain_request_status.html

```diff
@@ -41,8 +41,7 @@ <h1>Domain request for {{ DomainRequest.requested_domain.name }}</h1>
       </div>
     </div>
     <br>
-    <p> <b class="review__step__name">Last updated:</b> {{DomainRequest.updated_at|date:"F j, Y"}}<br>
-    <b class="review__step__name">Request #:</b> {{DomainRequest.id}}</p>
+    <p> <b class="review__step__name">Last updated:</b> {{DomainRequest.updated_at|date:"F j, Y"}}<br></p>
```

Comment:
`<p><b class="review__step__name">Last updated:</b> {{DomainRequest.updated_at|date:"F j, Y"}}</p>`

You should no longer need that br. Test and make sure.